astropy/table/table.py

```diff
@@ -2132,6 +2132,43 @@ def remove_rows(self, row_specifier):
         if hasattr(self, '_groups'):
             del self._groups
 
+    def iterrows(self, col_names=None):
```

Comment:
Call this arg `names` and make it a required argument.  Also, it should be `*names` here to match the API that I mentioned.